src/frontend/src/pages/SettingsPage/UserSettings/UserSettings.tsx

```diff
@@ -23,20 +24,49 @@ interface UserSettingsProps {
 export interface FormInput {
   defaultTheme: ThemeName;
   slackId: string;
+  city: string;
+  street: string;
+  state: string;
+  zipcode: string;
+  phoneNumber: string;
+  nuid: string;
 }
 
 const UserSettings: React.FC<UserSettingsProps> = ({ userId }) => {
   const [edit, setEdit] = useState(false);
   const { isLoading, isError, error, data: userSettingsData } = useSingleUserSettings(userId);
-  const update = useUpdateUserSettings();
+  const {
+    mutateAsync: updateUserSettings,
+    isLoading: updateUserSettingsIsLoading,
+    isError: updateUserSettingsIsError,
+    error: updateUserSettingsError
+  } = useUpdateUserSettings();
+  const toast = useToast();
 
-  if (isLoading || !userSettingsData || update.isLoading) return <LoadingIndicator />;
+  if (isLoading || !userSettingsData || updateUserSettingsIsLoading) return <LoadingIndicator />;
   if (isError) return <ErrorPage error={error} message={error.message} />;
-  if (update.isError) return <ErrorPage error={update.error!} message={update.error?.message!} />;
+  if (updateUserSettingsIsError) return <ErrorPage error={updateUserSettingsError!} />;
 
-  const handleConfirm = async ({ defaultTheme, slackId }: FormInput) => {
+  const handleConfirm = async ({ defaultTheme, slackId, street, city, state, zipcode, phoneNumber, nuid }: FormInput) => {
```

Comment:
<img width="875" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/005f63ae-05e9-40ea-9224-c8513e8fc792">
fAddress; also you should be able to change your slack id or theme without adding the secure settings, bc not everyone will need to add that info, and they shouldn't have to if they don't need to.

We should j make it a separate hook and a separate form/ page block right under User Settings called User Info or User Personal Info (my b for not catching this sooner)

Reply:
oof ok

Reply:
can we just keep i tin the same form but then just ignore any errors that get thrown for the secure settings

Reply:
Nah, I don't like that, it's messy. It also makes more sense to have this personal info in it's own section cause it is different from like Theme or Slack ID, so much so that we put it in its own model in the db.

Reply:
ok

Reply:
If you wanna j get a PR up for that other ticket, I don't mind swapping this ticket over tn or tmrw

Reply:
no ill do this quickly